assets/scss/1_base/2-base.scss

```diff
@@ -0,0 +1,37 @@
+$color-primary: #151515;
+$green: #92c064;
+$green-hover: #a9cc88;
+$red: #eb8d70;
+$red-hover: #f7c6b7;
+$white: #fff;
+$gray: #efefef;
+$strongGray: #7e7e7e;
+
+body {
+  background: #fdfdfd;
+  color: $color-primary;
+}
+
+.container {
+  max-width: 1160px;
+  width: 100%;
+  padding: 0 20px;
+  margin: 0 auto;
+}
+
+section {
+  margin-top: 40px;
+
+  @media all and (min-width: 600px) {
+    margin-top: 70px;
+
+    &::first-child {
```

Comment:
Double colons are used for pseudo-elements, pseudoselectors should have a single colon ;)